sdk/agrifood/agrifood-farming-rest/package.json

```diff
@@ -86,7 +86,7 @@
   "dependencies": {
     "@azure/core-auth": "^1.3.0",
     "@azure-rest/core-client": "1.0.0-beta.4",
-    "@azure/core-rest-pipeline": "^1.0.3",
+    "@azure/core-rest-pipeline": "1.1.0",
```

Comment:
Should this be `^1.1.0` (i.e. a caret range)?

Reply:
Ahh I missed `--caret` in the command. Thank you!